src/renderer/views/Watch/Watch.js

```diff
@@ -1266,6 +1264,23 @@ export default defineComponent({
       ]
     },
 
+    getAdaptiveFormatsInvidious: async function(existingInfoResult = null) {
+      let getInfoPromise = Promise.resolve(existingInfoResult)
+      if (existingInfoResult == null) {
+        getInfoPromise = invidiousGetVideoInformation(this.videoId)
+      }
+      const result = await getInfoPromise
+
+      return filterInvidiousFormats(result.adaptiveFormats, this.allowDashAv1Formats)
+        .map((format) => {
+          format.bitrate = parseInt(format.bitrate)
+          if (typeof format.resolution !== 'undefined') {
+            format.height = parseInt(format.resolution.replace('p', ''))
+          }
```

Comment:
Instead of doing a negative check for undefined you could do a positive check for a string.
```suggestion
          if (typeof format.resolution === 'string') {
            format.height = parseInt(format.resolution.replace('p', ''))
          }
```

Reply:
Yup, not sure why I forgot